docs/programming.md

```diff
@@ -57,3 +57,6 @@ Git and GitHub](https://journals.plos.org/ploscompbiol/article?id=10.1371/journa
 ## R
 * [Programming with R](https://neurohackademy.org/course/programming-with-r/) by Jeanette Mumford within Neurohackweek 2016
 * [Introduction to Neurohacking in R](https://www.coursera.org/learn/neurohacking) MOOC on coursera
+
+## Introduction & Basics
+- [Beginners coureswork on programmming, algorithms, and more](Khan Academy)
```

Comment:
@jesparent @Remi-Gau  How's this? Just the link to the "main" page of the website.
```suggestion
[Khan Academy] (https://www.khanacademy.org/): Beginners' coursework on programming, algorithms, and more.
```